hasher-matcher-actioner/hmalib/models.py

```diff
@@ -38,6 +39,33 @@ def remove_content_key_prefix(key: str) -> str:
         return key[len(DynamoDBItem.CONTENT_KEY_PREFIX) :]
 
 
+class SNSMessage:
+    def to_sns_message(self) -> str:
+        raise NotImplementedError
+
+    @classmethod
+    def from_sns_message(cls, message: str) -> "SNSMessage":
+        raise NotImplementedError
+
+
+@dataclass
+class Label:
+    key: str
+    value: str
+
+    def __post_init__(self):
+        """
+        Validate that the key name does not have colons.
+        """
```

Comment:
Is this a TODO or is the function meant to be blank? If so, i've often seen `pass` used in python

Reply:
That's an oopsie! Wonder how it got through mypy. Will fix.